### ISSUE TITLE
Live Lambda in VPC with AppSync

### ISSUE THREAD

**Issue by refactorthis** [anomalyco/sst#5218]
I've not been successful in getting Live mode to work with Lambdas in a VPC since the migration to AppSync.
I've created a VPC Interface for AppSync, and I can see the connection is successful from the bridge lambda.

The issue comes when publishing the message onto AppSync. My logs show the message is never received in AppSync, but I get no error in the bridge.

The only error is the eventual timeout when it receives no response within 3 seconds.

So essentially:
 * The bridge can connect to AppSync via the VPC interface
 * I do not see the published message in AppSync when the lambda is invoked via the runtime api.
 
 Any ideas what this could be?

**Comment by thdxr** [comment]
does your lambda have internet access?

**Comment by refactorthis** [comment]
No it does not. Does it need Internet Access? It has private vpc link to AppSync.

**Comment by thdxr** [comment]
yeah it does need internet access - i would maybe conditionally put the lambda outside the vpc in `$dev` mode

**Comment by refactorthis** [comment]
Can you explain why it requires Internet access? Isn't all messaging done
via AppSync pub/sub?
